product_currency/README.rst

```diff
@@ -25,23 +31,22 @@ To configure this module, you need to:
 #. No configuration nedeed.
 
 
-.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
-   :alt: Try me on Runbot
-   :target: https://runbot.adhoc.com.ar/
+Usage
+=====
 
-.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
-.. branch is "8.0" for example
+To use this module, you need to:
 
-Known issues / Roadmap
-======================
+#. In product view set "for currency" field and the currency of the product change to that set it in the force currency.
```

Comment:
> In product view set "for currency" fiel

misspelling error
"Force Currency"

Reply:
Also we could add something like this to the Usage section


* If you want to force a currency different from the company which the product is, then set this field.
* If you want to use the same currency of the company then just remove the value in "Force Currency" field.